### ISSUE TITLE
[BUG] Simple Example fails due to together import

### ISSUE THREAD

**Issue by evelynmitchell** [kyegomez/swarms#658]
**Describe the bug**
A fresh install in a colab notebook of the most recent swarms:
```
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
[<ipython-input-23-14444884eff9>](https://localhost:8080/#) in <cell line: 1>()
----> 1 from swarms import Agent
      2 
      3 agent = Agent(
      4     agent_name="Stock-Analysis-Agent",
      5     model_name="gpt-4o-mini",

8 frames
[/content/swarms/swarms/__init__.py](https://localhost:8080/#) in <module>
     11 
     12 # Import telemetry functions with error handling
---> 13 from swarms.telemetry.bootup import bootup  # noqa: E402, F403
     14 from swarms.telemetry.sentry_active import (  # noqa: E402
     15     activate_sentry,

[/content/swarms/swarms/telemetry/bootup.py](https://localhost:8080/#) in <module>
      4 from concurrent.futures import ThreadPoolExecutor
      5 
----> 6 from swarms.telemetry.auto_upgrade_swarms import auto_update
      7 from swarms.utils.disable_logging import disable_logging
      8 

[/content/swarms/swarms/telemetry/auto_upgrade_swarms.py](https://localhost:8080/#) in <module>
      2 import subprocess
      3 
----> 4 from swarms.utils.loguru_logger import initialize_logger
      5 from swarms.telemetry.check_update import check_for_update
      6 

[/content/swarms/swarms/utils/__init__.py](https://localhost:8080/#) in <module>
     14 )
     15 from swarms.utils.markdown_message import display_markdown_message
---> 16 from swarms.tools.prebuilt.math_eval import math_eval
     17 from swarms.utils.parse_code import extract_code_from_markdown
     18 from swarms.utils.pdf_to_text import pdf_to_text

[/content/swarms/swarms/tools/__init__.py](https://localhost:8080/#) in <module>
     23 from swarms.tools.openai_tool_creator_decorator import tool
     24 from swarms.tools.base_tool import BaseTool
---> 25 from swarms.tools.prebuilt import *  # noqa: F403
     26 from swarms.tools.cohere_func_call_schema import (
     27     CohereFuncSchema,

[/content/swarms/swarms/tools/prebuilt/__init__.py](https://localhost:8080/#) in <module>
      1 from swarms.tools.prebuilt.math_eval import math_eval
----> 2 from swarms.tools.prebuilt.code_executor import CodeExecutor
      3 
      4 __all__ = [
      5     "math_eval",

[/content/swarms/swarms/tools/prebuilt/code_executor.py](https://localhost:8080/#) in <module>
      2 import subprocess
      3 from loguru import logger
----> 4 from swarm_models.tiktoken_wrapper import TikTokenizer
      5 
      6 

[/usr/local/lib/python3.10/dist-packages/swarm_models/__init__.py](https://localhost:8080/#) in <module>
     39 from swarm_models.sam_two import GroundedSAMTwo
     40 from swarm_models.utils import *  # NOQA
---> 41 from swarm_models.together_llm import TogetherLLM
     42 from swarm_models.lite_llm_model import LiteLLM
     43 

[/usr/local/lib/python3.10/dist-packages/swarm_models/together_llm.py](https://localhost:8080/#) in <module>
      3 import loguru
      4 from dotenv import load_dotenv
----> 5 from together import Together
      6 
      7 load_dotenv()

ImportError: cannot import name 'Together' from 'together' (/usr/local/lib/python3.10/dist-packages/together/__init__.py)

---------------------------------------------------------------------------
NOTE: If your import is failing due to a missing package, you can
manually install dependencies using either !pip or !apt.

To view examples of installing some common dependencies, click the
"Open Examples" button below.
---------------------------------------------------------------------------
```

The code I was running:
```
from swarms import Agent

agent = Agent(
    agent_name="Stock-Analysis-Agent",
    model_name="gpt-4o-mini",
    max_loops="auto",
    interactive=True,
    streaming_on=True,
)

agent.run("What is the current market trend for tech stocks?")
```

This looks to be a bug in  
https://github.com/The-Swarm-Corporation/swarm-models/blob/737cf8f1115664bfabf311b9a5f3f9f89fd065fd/swarm_models/together_llm.py

Opening issue there also.

## Upvote & Fund

- We're using [Polar.sh](https://polar.sh/kyegomez) so you can upvote and help fund this issue.
- We receive the funding once the issue is completed & confirmed by you.
- Thank you in advance for helping prioritize & fund our backlog.

<a href="https://polar.sh/kyegomez/swarms/issues/658">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/api/github/kyegomez/swarms/issues/658/pledge.svg?darkmode=1">
  <img alt="Fund with Polar" src="https://polar.sh/api/github/kyegomez/swarms/issues/658/pledge.svg">
</picture>
</a>

**Comment by evelynmitchell** [comment]
Will need to be fixed by https://github.com/The-Swarm-Corporation/swarm-models/issues/11

**Comment by evelynmitchell** [comment]
This looks resolved in 6.4.9 Closing.
